### DISCUSSION TITLE
Has anyone been able to dump the famicom mighty bomb jack?

### DISCUSSION THREAD

**Discussion by fakkuyuu** [sanni/cartreader#792] [Need help]
Only really random CRC

**Comment by splash5** [comment]
![IMG_5270 2](https://github.com/sanni/cartreader/assets/2173030/0c150a7f-5999-4e54-a6ea-953fd1aed7f5)
![IMG_5271](https://github.com/sanni/cartreader/assets/2173030/7c143030-d42f-410d-9791-4e7759fdcfd3)

**Reply by fakkuyuu** [reply]
thanks

**Comment by fakkuyuu** [comment]
Haven't tested it on console yet. It may be because of the corrosion-like color in the resist layer.
![IMG_20230522_210439](https://github.com/sanni/cartreader/assets/120779915/c30fe079-ab18-45ec-8635-965dca19e4a7)

**Reply by fakkuyuu** [reply]
I found a part that was corroded and disconnected. I fixed it and dumped it successfully
![IMG_20230527_142124](https://github.com/sanni/cartreader/assets/120779915/9b87764d-e932-4b0a-ab29-7ef6503cd27e)
![IMG_20230527_145739](https://github.com/sanni/cartreader/assets/120779915/da0a93af-d4c1-41c0-a018-951875f88afd)
